prov/shm/src/smr_attr.c

```diff
@@ -61,7 +63,7 @@ struct fi_rx_attr smr_rx_attr = {
 };
 
 struct fi_tx_attr smr_hmem_tx_attr = {
-	.caps = SMR_TX_CAPS | FI_HMEM,
+	.caps = SMR_HMEM_TX_CAPS | FI_HMEM,
```

Comment:
Why does *HMEM* TX/RX caps not actually include FI_HMEM capability?  Seems rather odd.

Reply:
I think probably because at the time it made the code easier :) But you're right that doesn't make sense...
Now that they're separated, it makes sense to add it in. Will update

Reply:
@shefty do you mind if it's in the same patch?

Reply:
Cool. Updated.